tests/integration-tests/tests/performance_tests/test_openfoam.py

```diff
@@ -0,0 +1,82 @@
+import json
+import logging
+
+import boto3
+import pytest
+from assertpy import assert_that
+from remote_command_executor import RemoteCommandExecutionError, RemoteCommandExecutor
+
+# timeout in seconds
+OPENFOAM_INSTALLATION_TIMEOUT = 1800
+OPENFOAM_JOB_TIMEOUT = 600
+TASK_VCPUS = 36  # vCPUs are cut in a half because multithreading is disabled
+BASELINE_CLUSTER_SIZE_ELAPSED_SECONDS = {8: 742, 16: 376, 32: 185}
+PERF_TEST_DIFFERENCE_TOLERANCE = 5
+
+
+def perf_test_difference(perf_test_result, number_of_nodes):
```

Comment:
This is a function tha is used also in StarCCM+.
In the next iteration let's try to avoid code duplication and create a perf_test_common.py script with commond fuctions.